osx/KA-Lite-Monitor/KA-Lite-Monitor/AppDelegate.m

```diff
@@ -994,8 +994,11 @@ -(BOOL)resetApp {
     
     // Delete/reset the user preferences.
     [self resetPreferences];
-    
-    showNotification(@"Done resetting the app.  Please click the Apply button to repeat the install process.");
+    if(result){
```

Comment:
@amodia Your if-else construct must use spaces properly, this is hard to read.

Use proper coding convention for C-based languages like: https://gist.github.com/soffes/812796#control-structures.